apps/web/pages/getting-started/[[...step]].tsx

```diff
@@ -1,39 +1,82 @@
+import { getWorldViewSVGStr } from "@pages/getting-started/worldviewUtils";
+import type { ICountry } from "country-state-city";
+import { Country } from "country-state-city";
+import * as d3 from "d3";
```

Comment:
Between these two we're adding like 400kb to the JS bundle size

<img width="703" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/92b46aa0-7062-43a4-96a9-8d44827e1e04">

I would recommend lazy loading these components in the client.

Reply:
Importing only the necessary method helps by a lot as well

<img width="672" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/f516de2a-eb2b-4077-af4f-efd83d5033a9">